app/views/catalog/_accordion_section_library.html.erb

```diff
@@ -17,7 +17,9 @@
       </span>
     </div>
     <div class="details <%= id %>-location" aria-expanded="false">
-      <%= render :partial => "catalog/index_location", :locals => { :document => document } %>
+      <div data-controller="live-lookup" data-live-lookup-url-value="<%= availability_index_path %>">
```

Comment:
Can these `data-*` properties move to the parent div? Then we don't need a new div.